### PR TITLE
Fix `hide` -> `src`

### PR DESCRIPTION
**Pull request by t-bltg**

### CODE REVIEW

File: src/PlotDocs.jl [hunk 1]
```diff
@@ -79,15 +79,15 @@ function generate_cards(pkgname::Symbol; skip = get(Plots._backend_skips, pkgnam
             write(jl, "Plots.reset_defaults()  # hide\n")
         end
         # DemoCards use Literate.jl syntax with extra leading `#` as markdown lines
-        write(jl, "# $(replace(example.desc, "\n" => "\n  # hide"))\n")
+        write(jl, "# $(replace(example.desc, "\n" => "\n  # "))\n")
         for expr in example.exprs
             pretty_print_expr(jl, expr)
         end
-        write(jl, "\nmkpath(\"assets\")  # hide\n")
+        write(jl, "\nmkpath(\"assets\")  # src\n")
```
Comment (johnnychen94) [inline src/PlotDocs.jl:86]: Need to be `#src` without space.

https://fredrikekre.github.io/Literate.jl/v2/fileformat/#Syntax

Comment (t-bltg) [reply src/PlotDocs.jl:86]: This is way too confusing.

Comment (johnnychen94) [reply src/PlotDocs.jl]: Because `#src` is Literate syntax and `# hide` is Documenter syntax...

Comment (t-bltg) [reply src/PlotDocs.jl]: Terrible design decisions :grimacing:

**Pull request merged**